release/release_tests.yaml

```diff
@@ -112,6 +112,165 @@
 
   alert: default
 
+############################
+# Batch Inference Benchmarks
+############################
+
+# 10 GB image classification raw images with 1 GPU.
+# 1 g4dn.4xlarge
+- name: air_benchmark_torch_batch_prediction_1_gpu_10gb_parquet
```

Comment:
do we still want to keep (10GB, 1GPU), given we already have (300GB, 16GPU) to run nightly?

Reply:
I think it can help...it's a pretty cheap job cost wise and can help isolate whether regressions are occurring in the single node/single GPU case, or only in the multi node case. But I am open to removing it.